### ISSUE TITLE
Use of isa/UNIVERSAL removed in latest perl5 (5.22 on ubuntu 16.04)

### ISSUE THREAD

**Issue by gdevenyi** [BIC-MNI/ILT#4]
This function is used in a couple of places in ILT, it's been giving a warning before, on new systems it doesn't work at all

https://stackoverflow.com/questions/35258993/perl-universal-does-not-export-anything

**Comment by nathankchan** [comment]
Issue still exists on perl 5.28. The command that throws the error is `create_verify_image`. 

Seems like this is due to deprecated syntax where `use UNIVERSAL qw(isa)` was used. Alas, I'm not familiar enough with perl to propose a solution.

Error log from QC stage of [minc-bpipe-library](https://github.com/CobraLab/minc-bpipe-library):

```
UNIVERSAL does not export anything at /quarantine/MINC/1.9.16/perl/ILT/ProgUtils.pm line 7.
BEGIN failed--compilation aborted at /quarantine/MINC/1.9.16/perl/ILT/ProgUtils.pm line 7.
Compilation failed in require at /quarantine/MINC/1.9.16/perl/ILT/LayoutUtils.pm line 6.
BEGIN failed--compilation aborted at /quarantine/MINC/1.9.16/perl/ILT/LayoutUtils.pm line 6.
Compilation failed in require at /quarantine/MINC/1.9.16/perl/ILT/ImageLayout.pm line 34.
BEGIN failed--compilation aborted at /quarantine/MINC/1.9.16/perl/ILT/ImageLayout.pm line 34.
Compilation failed in require at /quarantine/MINC/1.9.16/perl/ILT/LayoutInclude.pm line 35.
BEGIN failed--compilation aborted at /quarantine/MINC/1.9.16/perl/ILT/LayoutInclude.pm line 35.
Compilation failed in require at /quarantine/MINC/1.9.16/bin/create_verify_image line 33.
BEGIN failed--compilation aborted at /quarantine/MINC/1.9.16/bin/create_verify_image line 33.
```

**Comment by gdevenyi** [comment]
Fixed by #5
